src/components/left-navigation.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Offcanvas, Nav } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import "./navigation.css";
+import { FaUserFriends } from "react-icons/fa";
+import { FaHouseChimney } from "react-icons/fa6";
+import { BsFillFileEarmarkBarGraphFill } from "react-icons/bs";
+import { HiUserGroup } from "react-icons/hi";
+
+function LeftNavigation({ show, onHide }) {
+  return (
+    <Offcanvas show={show} onHide={onHide} backdrop="static">
+      <Offcanvas.Header closeButton>
+        <Offcanvas.Title></Offcanvas.Title>
+      </Offcanvas.Header>
+      <Offcanvas.Body>
+        <Nav defaultActiveKey="/home" className="flex-column">
+          <Nav.Link href="/home">
+            <FaHouseChimney style={{ fontSize: "35px" }} />
+            Home
+          </Nav.Link>
+          <Nav.Link as={Link} to="/classes">
+            <HiUserGroup style={{ fontSize: "35px" }} />
```

Comment:
we should try to avoid in-line styling like this as much as possible. If needed, create a css class.

If we need to change the fontSize from 35 to 40 for example, we need to change in multiple location. If we have it in class, we only need to change once.